pkg/deploy/local/local_provider.go

```diff
@@ -20,98 +26,166 @@ import (
 	wasmeutil "github.com/solo-io/wasme/pkg/util"
 )
 
-type Provider struct {
+type Runner struct {
 	Ctx context.Context
 
-	// input config
+	// input bootstrap config
 	Input io.ReadCloser
 
-	// path to output file
-	OutFile string
+	// output config YAML only (DryRun), rather than invoking docker run
+	Output io.Writer
+
+	// path to root storage dir
+	// default is ~/.wasme/store
+	Store store.Store
 
-	// the destination for storing the filter on the local filesystem
-	FilterPath string
+	// additional args passed to the `docker run` command when running Envoy. Ignored if using DryRyn
+	DockerRunArgs []string
 
-	// Use JSON instead of YAML for config (defaults to false)
-	UseJsonConfig bool
+	// additional args passed to the `envoy` command when running Envoy in Docker. Ignored if using DryRyn
+	EnvoyArgs []string
+
+	// the image ref for Envoy to run with docker. Ignored if using DryRyn
+	EnvoyDockerImage string
 }
 
-func (p *Provider) getConfig() (*envoy_config_bootstrap_v2.Bootstrap, error) {
-	b, err := ioutil.ReadAll(p.Input)
+// applies the filter to all selected workloads in selected namespaces
```

Comment:
what does namespaces mean in this context?

Reply:
fixed comment